web/backto.go

```diff
@@ -0,0 +1,43 @@
+package web
+
+import (
+	"net/url"
+	"strings"
+)
+
+var hostnameWhitelist = map[string]bool{
+	"localhost": true,
+	"viam.dev":  true,
+	"viam.com":  true,
```

Comment:
should these be app.viam.dev and app.viam.com?

Reply:
I left them as this because we have toyed with allowing users to log in from the marketing page and docs, but maybe I am being a little too forward-thinking here 😃 .

Let's start stricter; I'll update it.

Reply:
gotcha! yeah i wasnt sure if it would pass the `hostnameWhitelist[hostname]` check for `app.viam.com` urls otherwise. could be misreading though